src/main/java/seedu/trackbeau/model/customer/SearchContainsKeywordsPredicate.java

```diff
@@ -59,8 +61,21 @@ public boolean test(Customer customer) {
 
     @Override
     public boolean equals(Object other) {
-        return other == this // short circuit if same object
-                || (other instanceof SearchContainsKeywordsPredicate // instanceof handles nulls
-                && keywords.equals(((SearchContainsKeywordsPredicate) other).keywords)); // state check
+        if (other == this) {
```

Comment:
Similarly, for lines 64-79 seems to have arrow-head style code, maybe you can try doing
`if (other == this) { return true; }`
`if !(other instanceof SearchContainsKeywordsPredicate) { return false; }`
`for (int i = 0; i < 9; i++) { ... }`

Reply:
okay will do